index.js

```diff
@@ -76,9 +76,11 @@ class MustacheMailer {
     const textPath = this._resolveTemplateFile(name + '.text.hbs', files)
     const metaPath = this._resolveTemplateFile(name + '.meta.hbs', files)
 
-    const [textT, htmlT, metaT] = await Promise.map(
-      [textPath, htmlPath, metaPath],
-      this._loadTemplate)
+    const [textT, htmlT, metaT] = await Promise.all([
+      this._loadTemplate(textPath, {noEscape: true}),
```

Comment:
Disable HTML escaping for text emails. This is to fix visible HTML entity codes in npm welcome emails:

<img width="301" alt="screen shot 2018-10-25 at 14 02 14" src="https://user-images.githubusercontent.com/1410204/47502191-13c57300-d85f-11e8-8bc4-88de328d6cce.png">